lib/dependencies/HarmonyModulesHelpers.js

```diff
@@ -68,3 +68,15 @@ HarmonyModulesHelpers.getActiveExports = function(module) {
 		return arr;
 	}, [])
 };
+
+//check if an harmony module has been required by CommonJS module
+HarmonyModulesHelpers.isRequiredByCommonJs = function(module) {
+	var reasons = module.reasons;
+	for(var i = 0; i < reasons.length; i++) {
+		var reason = reasons[i];
+		if(reason.dependency && reason.dependency.constructor.name === 'CommonJsRequireDependency') {
```

Comment:
I just found this way to check if it's been required by commonjs. Is there a better way to do it? 

I will certainly look for better way to do it too.